framework/include/eventloop/eventloop.h

```diff
@@ -89,6 +89,27 @@ el_timer_t *eventloop_add_timer(unsigned int timeout, bool repeat, timeout_callb
  */
 int eventloop_delete_timer(el_timer_t *timer);
 
+/**
+ * @brief Create timer to call the function in the given period of time asynchronously
+ * @details @b #include <eventloop/eventloop.h> \n
+ * This API is same as eventloop_add_timer(), but it yields the event to evnet loop task.\n
+ * So you don't need to call eventloop_loop_run() to run loop. \n
+ * In other words, the timer event is added to a loop handled by eventloop task, \n
+ * and then registered callback is executed in the context of eventloop task asynchronously, not task which created timer.
+ * @param[in] timeout interval in milliseconds from the current time of loop for calling callback function
+ * @param[in] repeat the value which represents whether timer runs repeatly or not. \n
+ *            You can stop the periodic timer by calling eventloop_delete_timer
+ * @param[in] func the callback function to be called
+ * @param[in] data data to pass to func when func is called
+ * @return On success, A pointer of created timer is returned if 'repeat' is true. \n
+ *         You should free used timer resources after using timer by calling eventloop_delete_timer with returned pointer. \n
+ *         On the other hand, NULL is returned if 'repeat' is false. \n
+ *         So you don't have to do anything because all resources are freed internally after callback execution. \n
+ *         On failure, NULL is returned
```

Comment:
How to discriminate two cases: returning NULL on failure and returning NULL on success when repeat is false?

Reply:
Both cases return NULL, so they can distinguished by enabling debug.